.github/workflows/reusable-build-win-32.yml

```diff
@@ -0,0 +1,27 @@
+# gh-84728: "paths-ignore" is not used to skip documentation-only PRs, because
+# it prevents to mark a job as mandatory. A PR cannot be merged if a job is
+# mandatory but not scheduled because of "paths-ignore".
```

Comment:
Remove this comment, it's not valid for the reusable workflow :)

Reply:
fixed fixed -> https://github.com/jota12x/cpython/pull/2/commits/510ab5637279ac4bb3c0fc76bc4dd79b71b42eb3